src/tests/ftest/io/llnl_mpi4py.py

```diff
@@ -0,0 +1,157 @@
+#!/usr/bin/python
+'''
+    (C) Copyright 2019 Intel Corporation.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+    GOVERNMENT LICENSE RIGHTS-OPEN SOURCE SOFTWARE
+    The Government's rights to use, modify, reproduce, release, perform,
+    display, or disclose this software are subject to the terms of the Apache
+    License as provided in Contract No. B609815.
+    Any reproduction of computer software, computer software documentation, or
+    portions thereof marked with this legend must also reproduce the markings.
+    '''
+
+from __future__    import print_function
+import os
+import sys
+import json
+from avocado       import Test
+
+sys.path.append('./util')
+sys.path.append('../util')
+sys.path.append('../../../utils/py')
+sys.path.append('./../../utils/py')
+import ServerUtils
+import WriteHostFile
+from MpioUtils import MpioUtils, MpioFailed
+from daos_api import DaosContext, DaosPool, DaosApiError
+
+class LlnlMpi4py(Test):
+    """
+    Runs LLNL and MPI4PY test suites.
+    """
+
+    def __init__(self, *args, **kwargs):
```

Comment:
We've determined that __init__ and setup are both allowed names for the "constructor" is there some benefit in breaking out the variable initialization?  Just put them at the top of setup seems simpler.